src/main/java/com/esri/samples/map/read_geopackage/README.md

```diff
@@ -0,0 +1,41 @@
+<h1>Read GeoPackage</h1>
+
+<p>Demonstrates how to read rasters and feature tables from geoPackages to show them as layers in a map.</p>
+
+<p><img src="ReadGeoPackage.png" /></p>
+
+<h2>How to use the sample</h2>
+
+<p>The layers in the geoPackage, which have not been added to the map are shown in the bottom list. Click an item to 
+show it as a layer in the map. Layers in the map are listed in the top list. Click layers from the top list to 
+remove them from the map.</p>
+
+<h2>How it works</h2>
+
+<p>To read layers from a geoPackage and show them in a map:</p>
+
+<ol>
+    <li>Create a <code>GeoPackage</code> with the path to the local geoPackage file.</li>
+    <li>Load the <code>GeoPackage</code> with <code>GeoPackage.loadAsync</code>.</li>
+    <li>When it's done loading, get the <code>GeoPackageFeatureTable</code>s inside with <code>geoPackage
+    .getGeoPackageFeatureTables()</code>.</li>
+    <li>For each feature table, create a feature layer with <code>new FeatureLayer(featureTable)</code>.</li>
```

Comment:
Would either switch Raster Layer step to come first or change Feature Layer step to come first in the code. This way when someone is going through the code and this it flows together.